lib/katello/tasks/reset.rake

```diff
@@ -1,29 +1,48 @@
 namespace :katello do
   namespace :reset_backends do
-    service_stop = "sudo /sbin/service %s status > /dev/null && sudo /sbin/service %s stop"
-    service_start = "sudo /sbin/service %s start"
+    service_stop = "sudo /usr/bin/systemctl status %s > /dev/null && sudo /usr/bin/systemctl stop %s"
+    service_start = "sudo /usr/bin/systemctl start %s"
```

Comment:
Any reason to not use foreman-maintain here?

Reply:
Right now foreman-maintain does not know about the pulp3 services when I was seeing what restarts on the dev box. Once it does I am happy to change it over to that.

Reply:
Ah! makes sense